### DISCUSSION TITLE
http-missing-security-headers 建议删除

### DISCUSSION THREAD

**Discussion by zan8in** [projectdiscovery/nuclei#1521] [Q&A] [answered]
http-missing-security-headers 建议删除，该漏洞没有重要意义，并且在批量扫描URL时候容易造成结果污染。

**Comment by No-Github** [comment, marked as answer]
这个偏向于安全配置检测，不属于漏洞扫描，如果不需要，建议在运行时使用 -et 选项进行排除。
